test/levy_areas.jl

```diff
@@ -84,7 +84,7 @@ Problem 2.3.3 from
 Kloeden, P. E., Platen, E., & Schurz, H. Numerical solution of SDE through computer
 experiments. Springer Science & Business Media. (2012)
 """
-function test_path_convergence(Wik, dt = 1.0, ps = [Int(1e1),Int(1e3),Int(1e6)])  
+function test_path_convergence(Wik, dt = 1.0, ps = [Int(2e1),Int(1e3),Int(1e6)])  
```

Comment:
This doesn't work because of the order in which the random numbers are generated. It is *not* the case that two different values of `p` share the first few random numbers in the same order.

Reply:
Next, I planned to average over some seeds and see if this eliminates the errors that sometimes `p=1e3` could be worse than `p=1e2`.

Reply:
> I agree that this is not generally the case. However, shouldn't this work once I fix the seed? So, I force that the larger `p` values contain the random numbers of smaller `p` values
> 
> ```
> Random.seed!(seed)
> I = StochasticDiffEq.get_iterated_I(dt, W.dW, W.dZ, Wik, p, 1)
> ```

That's exactly what doesn't work. There are different types of random numbers needed in the algorithm, lets call them for simplicity `a_1` to `a_p`, `b_1` to `b_p` and `c`.
If `p` is 4 and our random number generator (again for simplicity) generates increasing integers, we get:
`a = 1 2 3 4`
`b = 5 6 7 8`
and `c = 9`.
But for `p` equal to 5 we get:
`a = 1 2 3 4 5`
`b = 6 7 8 9 10`
and `c = 11`.
Thus they don't share the values because of the order in which they are generated/needed.